packages/story-editor/src/app/story/useStoryReducer/test/setBackgroundElement.js

```diff
@@ -29,7 +29,7 @@ describe('setBackgroundElement', () => {
           id: '111',
           elements: [
             { id: '000', isBackground: true },
-            { id: '123', opacity: 20, isBackground: false },
```

Comment:
Perhaps an opportunity/reason to convert the reducer tests to TS?

Reply:
They would have to become a lot more complex though, as they cheat and only use a tiny subset of the required properties. But they would also become a lot better, so it would be worth it for sure. It is however quite a task.